WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -330,6 +316,13 @@ on_retreat = {
 
 #Chinese traits
 son_of_heaven = {
+	potential = {
+		OR = {
+			culture_group = sinic
+			religion = confucian
+		}
+		higher_real_tier_than = KING
+	}
```

Comment:
The existing flavor does not exist. In the Chinease "VIET_ZHONGUGO" file are hundreds of entries with no correspondance in the code. For example, there is no Khitan Invasion despite it being in that list.